apps/web/util/helpers/s3-helpers.ts

```diff
@@ -29,10 +29,13 @@ export const createNewBucket = async (
 };
 
 export const beforeCreatingDoc = async (req: NextApiRequest, res: NextApiResponse, body: any) => {
+
   const { data, type } = body;
 
   switch (type) {
     case "firebase":
+    case "wasabi":
+    case "digitalocean":
```

Comment:
i think these two cases for "wasabi" and "digitalocean" will go with s3 and backblaze since we need to configure the CORS rules for them (unlike firebase)